src/app/routes/utils/fetchPageData/index.js

```diff
@@ -31,16 +31,18 @@ const fetchPageData = async ({
   path,
   timeout,
   shouldLogFetchTime = !onClient(),
+  options,
   ...loggerArgs
 }) => {
   const url = path.startsWith('http') ? path : getUrl(path);
   const effectiveTimeout = timeout || PRIMARY_DATA_TIMEOUT;
-  const fetchOptions = {
+  const defaultOptions = {
     headers: {
       'User-Agent': 'Simorgh/ws-web-rendering',
     },
     timeout: effectiveTimeout,
   };
+  const fetchOptions = options || defaultOptions;
```

Comment:
with options provided we still need to apply the timeout from here: https://github.com/bbc/simorgh/pull/9813/files#diff-2f6af00c0fdb21e32f5a4068954f1a455ae07ea0499aa290a8d8f9af89626f96L42 

Maybe we could pass these options in as individual parameters: https://github.com/bbc/simorgh/pull/9813/files#diff-6e0488cd2114bbba672b980609882f2d3f360a3cedb6732d5e2d5bb89faaf361R12-R16 ? At the moment passing in the options directly breaks the abstraction a bit on `fetchPageData`, with the timeout being removed and the `User-Agent` header being lost. We could then apply the parameters to the options in here respecting the timeout etc.